tests/functional/u/unsupported_delete_operation.rc

```diff
@@ -0,0 +1,2 @@
+[testoptions]
+requires = six
```

Comment:
Idem.
In fact those tests are those which are failing on the CI after PyCQA/astroid#841 has been merged...